app/src/main/res/values-fr/strings.xml

```diff
@@ -156,9 +158,15 @@
     <string name="settingsWebTrackingProtectionTitle">Protection contre le pistage sur le Web</string>
     <string name="settingsWebTrackingProtectionDescription">Activation par défaut</string>
     <string name="settingsHeadingSettings">Paramètres</string>
+    <string name="settingsHeadingMainSettings" tools:ignore="MissingTranslation">Réglages principaux</string>
```

Comment:
@lmac012  do you know why we have `tools:ignore=“MissingTranslation” `for all the new translations in this PR?

Reply:
They were meant to be translated at some point, but I misunderstood how things work with the translations. These should all be behind the feature flag.